utils/util_test.go

```diff
@@ -27,3 +28,35 @@ func TestBitsBytesConvert(t *testing.T) {
 		}
 	}
 }
+
+func TestIntToBytes(t *testing.T) {
+	numInt := 42
+	b := IntToBytes(numInt)
+	if int(binary.LittleEndian.Uint32(b)) != numInt {
+		t.Fatal("Conversion to bytes looks wrong!")
+	}
+	// TODO It is possible to call `IntToBytes` with a signed int < 0
```

Comment:
Continue the discussion in https://github.com/coniks-sys/coniks-go/pull/43#discussion_r73206002.

Do you have any explanation for this? I really don't know the reason why Golang developers do this. Even if I try `int(4294967254) == -42`, it still returns false.